.travis.yml

```diff
@@ -1,24 +1,24 @@
 os:
   - linux
-dist: trusty
+  - osx
 
 language: c
 
+env:
+  - BRANCH=0.19.4
+  - BRANCH=devel
```

Comment:
We aren't tied to the compiler anymore, why does it need to be tied to a commit? If nimble fails to build due to compiler change, better to fail early right?

Reply:
Because `devel` can break at any time, I don't want to get travis failures because of that.

Reply:
Might it make sense to eventually pin to nightlies since those only get posted when all tests pass? Need to add the feature to choosenim to download/install nightlies but might make things more stable.

Reply:
Done.